### PR TITLE
Test several cgal releases

### PR DESCRIPTION
**Pull request by xantares**

### CODE REVIEW

**Comment by maxGimeno** [comment]
Seems perfectly good to me.

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,12 @@ language: generic
 sudo: required
 dist: trusty
 
+matrix:
+  include:
```
Comment (maxGimeno) [inline .travis.yml]: No 'include:' here

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,12 @@ language: generic
...
+    - env: CGAL_VER=4.8
+    - env: CGAL_VER=4.9
+    - env: CGAL_VER=4.10
+
```
Comment (maxGimeno) [inline .travis.yml]: you can use env: matrix: instead of matrix: env -env -env... but I don't think it matters

File: .travis.yml [hunk 3]
```diff
@@ -2,6 +2,12 @@ language: generic
 sudo: required
 dist: trusty
 
+env:
+  matrix:
+    - CGAL_VER=4.8
+    - CGAL_VER=4.9
+#    - env: CGAL_VER=4.10
```
Comment (sloriot) [inline .travis.yml]: we can now test 4.10

**Comment by lrineau** [comment]
I have modified the .travis.yml file, to test CGAL-4.10.

**Comment by xantares** [comment]
I guess it can be merged now

**Pull request merged**